project-base/app/tests/App/Functional/Model/Pricing/RelevantSpecialPriceForProductTest.php

```diff
@@ -51,6 +51,13 @@ final class RelevantSpecialPriceForProductTest extends TransactionFunctionalTest
      */
     private readonly PriceListFacade $priceListFacade;
 
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->setInputPriceTypeToInputPriceTypeWithVat();
```

Comment:
do not we want another test for input price type without VAT?

Reply:
I did not feel the need, do you?